t/Inline.t

```diff
@@ -0,0 +1,24 @@
+# watch -*-perl-*-
+
+use strict;
+use Test::More tests => 4;
+use Event ();
+
+require Config;
+require Event::MakeMaker;
+
+my $path = $Event::MakeMaker::installsitearch;
```

Comment:
This all looks very fragile. It's enough to just call the `Inline` method with `C`, and then with anything else, and check each return-value `isnt` undef.

Reply:
will fix